app/src/main/java/org/mozilla/fenix/settings/SettingsFragment.kt

```diff
@@ -501,6 +502,8 @@ class SettingsFragment : PreferenceFragmentCompat() {
     private fun navigateUserToDefaultAppsSettings() {
         if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.N) {
             val intent = Intent(android.provider.Settings.ACTION_MANAGE_DEFAULT_APPS_SETTINGS)
+            intent.putExtra(SETTINGS_SELECT_OPTION_KEY, DEFAULT_BROWSER_APP_OPTION)
+            intent.putExtra(SETTINGS_SHOW_FRAGMENT_ARGS, bundleOf(SETTINGS_SELECT_OPTION_KEY to DEFAULT_BROWSER_APP_OPTION))
```

Comment:
Can this line be splitted on two rows?
Detekt fails because the 120 characters limit is exceeded.

Reply:
Just fixed it!